packages/twenty-server/src/database/commands/upgrade-version/0-23/0-23-migrate-link-fields-to-links.command.ts

```diff
@@ -225,6 +225,7 @@ export class MigrateLinkFieldsToLinksCommand extends CommandRunner {
               id: tmpNewLinksField.id,
               workspaceId: tmpNewLinksField.workspaceId,
               name: `${fieldName}`,
+              isCustom: false,
```

Comment:
when using the service, create endpoint overrides input value for isCustom, but update does not